v4/export/dump_test.go

```diff
@@ -195,3 +195,22 @@ func (s *testDumpSuite) TestDumpTableWhereClause(c *C) {
 	c.Assert(tbDataRes.Rows().HasNext(), IsFalse)
 	c.Assert(mock.ExpectationsWereMet(), IsNil)
 }
+
+func (s *testDumpSuite) TestDumpTableAllGeneratedColumns(c *C) {
+	mockConfig := DefaultConfig()
+	mockConfig.SortByPk = false
+	db, mock, err := sqlmock.New()
+	c.Assert(err, IsNil)
+	conn, err := db.Conn(context.Background())
+	c.Assert(err, IsNil)
+
+	showCreateTableResult := "CREATE TABLE a (a INT as (1))"
+	rows := mock.NewRows([]string{"Table", "Create Table"}).AddRow("a", showCreateTableResult)
```

Comment:
That error shows dumpling queried a SQL (which may triggered by other logic of dumpling), but `mock` was not configurated to receive that SQL. 

You may add `ExpectQuery` like these lines if you believe that's indeed an expected SQL.

Reply:
Thanks for your reply! I have solved the bug of unit test by adding more mock expectations.